test/conformance/enqueue/urEnqueueDeviceGlobalVariableRead.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) 2023 Intel Corporation
+// Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+// See LICENSE.TXT
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#include <uur/fixtures.h>
+
+using urEnqueueDeviceGetGlobalVariableReadTest = uur::urGlobalVariableTest;
+UUR_INSTANTIATE_DEVICE_TEST_SUITE_P(urEnqueueDeviceGetGlobalVariableReadTest);
+
+TEST_P(urEnqueueDeviceGetGlobalVariableReadTest, Success) {
```

Comment:
I've only included 1 success case since to assert behaviour you have to use both entry-points.